builtin/game/register.lua

```diff
@@ -177,7 +179,13 @@ function core.register_item(name, itemdef)
 	--core.log("Registering item: " .. itemdef.name)
 	core.registered_items[itemdef.name] = itemdef
 	core.registered_aliases[itemdef.name] = nil
-	register_item_raw(itemdef)
+
+	-- Do not allow mods to register ignore with the engine
+	if name ~= "ignore" then
+		register_item_raw(itemdef)
+	elseif is_overriding then
+		minetest.log("warning", "Attempted redefinition of \"ignore\"")
```

Comment:
use the `core` namespace in builtin.

Reply:
Please do.

Reply:
Done, 
l also changed the comment because avoiding mods to register to core is just a redundant side effect.